src/main/java/com/faforever/client/game/GameTileController.java

```diff
@@ -110,6 +112,8 @@ public void setGame(Game game) {
     ));
 
     lockIconLabel.visibleProperty().bind(game.passwordProtectedProperty());
+    friendsLabel.visibleProperty().bind(new SimpleBooleanProperty(!playerService.friendsInGame(game).isEmpty()));
+    gameCardRoot.setId(!playerService.friendsInGame(game).isEmpty() ? "friend" : "");
```

Comment:
That should not happen...Changes in style should be done via state change not changing ids... Look at how it is done elsewhere https://stackoverflow.com/questions/37072514/how-to-add-class-or-pseudoclass-programmatically-to-custom-control-in-javafx

Reply:
got it!